src/components/LandingPage/LandingPage.module.css

```diff
@@ -13,11 +12,10 @@
 }
 
 .LandingPageMainContent {
-  flex-grow: 1;
   display: flex;
-  flex-direction: row;
-  align-self: center;
+  justify-content: space-between;
   padding: 3rem 2rem 4rem;
```

Comment:
why is the top padding different from the bottom? also, make the left/write value `3rem` for consistency with the sections before and after.

Reply:
i believe `padding: 3rem ` is enough

Reply:
same applies to the `why`, `migrate`, and `explore` sections

Reply:
try to keep the consistency

Reply:
good job tho

Reply:
@kawerewagaba I have made the changes for this